web/styles.css

```diff
@@ -175,31 +166,119 @@ body {
 
 .slide-up-enter-active,
 .slide-up-leave-active {
-  transition: all 0.12s ease;
+    transition: all 0.12s ease !important;
```

Comment:
Gah so many importants, can we get rid of these?

Reply:
I want to so bad lol, but no, these transitions are lower priority than basically everything else for no good reason, that's just how vue does it. So this was the recommendation for making them work by both of the forums I went to